src/api.rs

```diff
@@ -140,6 +139,14 @@ impl Covercrypt {
             keep_old_secrets,
         )
     }
+
+    pub fn full_decaps(
+        &self,
+        msk: &MasterSecretKey,
+        enc: &XEnc,
+    ) -> Result<Vec<(Right, Secret<SHARED_SECRET_LENGTH>)>, Error> {
+        full_decaps( enc, msk)
```

Comment:
Did you run `cargo fmt`?
```suggestion
        full_decaps(enc, msk)
```